config/config.go

```diff
@@ -0,0 +1,46 @@
+package config
```

Comment:
I was considering growing it later, but if that ends up happening we can always move it then.